src/helpers.js

```diff
@@ -47,12 +48,13 @@ const getAndValidateParams = (paramsToCheck, asyncOperationDescriptor) => {
   const { logger } = asyncOperationManagerConfig.getConfig();
   if (asyncOperationDescriptor.requiredParams) {
     // make sure that every requiredParams is included in the asyncOperationParams object and that
-    // none of the values are falsey
-    if (!every(asyncOperationDescriptor.requiredParams, partial(has, asyncOperationParams)) || (asyncOperationParams && some(asyncOperationParams, paramValue => !paramValue))) {
+    // none of the values are undefined
+    if (!every(asyncOperationDescriptor.requiredParams, partial(has, asyncOperationParams)) ||
```

Comment:
I think Sam mentioned something about an api request requiring an undefined to be passed to grab all calendar appointments.  We may still want to allow undefined values.  Would the hasOwnProperty approach still work?

Reply:
It looks like we're already checking if the properties are there. Sam and I talked further about the `undefined` situation and he says that we're not actually using asyncOperationsManager in that case. We both felt that checking for properties but allowing their value to be `undefined` seems odd? If something is a required parameter, it should have SOME assigned value, even if that value is explicitly `null`.

Reply:
cool - im good with it then